### ISSUE TITLE
No Maxpool in VGG

### ISSUE THREAD

**Issue by kbui1993** [google-research/growneuron#12]
I looked at your VGG implementation for gradmax. Is there a reason why the MaxPool layers are omitted in your implementation?

**Comment by evcu** [comment]
It has been a while, but do VGG for cifar uses max-pool? Images are already quite small

**Comment by kbui1993** [comment]
The original VGG implementation did have max pool at the end of each block, but it was originally made for Imagenet instead of CIFAR. 

I'm trying to replicate your work in torch. It worked as expected for simple MLP and CNN model, but for VGG11, Gradmax hasn't made much of a difference in decreasing the training loss. I'm suspecting it may be because of the max pool layer, so that's why I asked. If you happen to remember the reason for omitting the max pool layer for your VGG implementation, please let me know.

**Comment by kbui1993** [comment]
I'm closing the issue. I realized that the actual issue with VGG is that it is overparametrized for CIFAR-10, so I had to reduce the size of it even further to see the impact of gradmax. 

Anyways, thank you for responding to my issue.
